### DISCUSSION TITLE
Creating New Profiles for Rules

### DISCUSSION THREAD

**Discussion by ZaViper** [sfmict/MountsJournal#27] [Ideas]
Thanks for adding all the new conditionals to the rules feature! I wanted to ask if it would be possible to create different profiles for different characters. I would like to use different rule sets for different characters to use. Trying to funnel all the different rules for different characters on one rule list is making the rule list really long and hard to navigate.

**Comment by sfmict** [comment]
Hi, I thought about it and yes, I can make it. It's definitely better than creating a condition with choosing a character, so it will probably appear :)

**Comment by sfmict** [comment]
Added :)

**Reply by ZaViper** [reply]
Thank you so much! :)
